proto/streamlit/proto/PlotlyChart.proto

```diff
@@ -30,6 +30,8 @@ message PlotlyChart {
   // If True, will overwrite the chart width spec to fit to container.
   bool use_container_width = 5;
 
+  string theme = 6;
```

Comment:
nit: maybe add a comment here (just for consistency since all other props in this proto have comments)